src/mol-gl/shader/cylinders.frag.ts

```diff
@@ -9,6 +9,7 @@ precision highp float;
 precision highp int;
 
 #define bumpEnabled
+#define multiColor
```

Comment:
Could this be a define that is only set for visuals that need it? Also, how about calling it `dualColor`?

Reply:
Done with last commit